sources/meta-yoe/conf/projects/var-som-mx8x/config.conf

```diff
@@ -0,0 +1,58 @@
+DISTRO = "yoe"
```

Comment:
@kraj FYI, it does seem to be working not to set MACHINE in this config (gets set in envsetup.sh). This might move us toward support multiple machines in some projects.